src/main/java/frc/trigon/robot/commands/CommandConstants.java

```diff
@@ -1,18 +1,22 @@
-package frc.trigon.robot.constants;
+package frc.trigon.robot.commands;
 
 import edu.wpi.first.math.geometry.Rotation2d;
 import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.util.Color;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.InstantCommand;
 import frc.trigon.robot.RobotContainer;
-import frc.trigon.robot.subsystems.ledstrip.LEDStripCommands;
-import frc.trigon.robot.subsystems.ledstrip.LEDStripConstants;
+import frc.trigon.robot.constants.OperatorConstants;
 import frc.trigon.robot.subsystems.swerve.SwerveCommands;
 import org.trigon.hardware.misc.XboxController;
+import org.trigon.hardware.misc.leds.LEDCommands;
+import org.trigon.hardware.misc.leds.LEDStrip;
 import org.trigon.utilities.mirrorable.MirrorablePose2d;
 
-import java.awt.*;
-
+/**
+ * A class that contains commands that run the exact same command each time.
+ * These are different from GeneralCommands because they don't need to be sent any parameters and always do the same thing.
```

Comment:
1. talk about command factories and not only general commands
2. the difference you mentioned isn't correct / fully explaining
3. `class that contains commands that run the exact same command each time` - the main idea and difference from the factories is that here you create commands that only need parameters, and there you have actual logic for commands